packages/react-components/react-utilities/src/compose/getSlotClassNameProp.ts

```diff
@@ -0,0 +1,9 @@
+import { SLOT_ORIGINAL_CLASS_NAME_SYMBOL } from '../compose/constants';
+import type { UnknownSlotProps } from '../compose/types';
+
+export const getSlotClassNameProp = (slot: UnknownSlotProps) => {
```

Comment:
could we use a more descriptive name?

```suggestion
export const getSlotOriginalClassNameProp = (slot: UnknownSlotProps) => {
```

Reply:
Renamed to `getSlotOriginalClassNameProp`, and leaving this thread open for further bikeshedding on the name :)

Reply:
I think I find the "original" part a bit misleading because that makes me think of the classnames provided by the control itself rather than the supplemental classname provided by the user. i.e. `getSlotOriginalClassNameProp` might return something like `fui-Button`.

Either that, or `getSlotOriginalClassNameProp` would return the final result of `mergeClasses` within the component (the "original" classname before the custom style hook was run).

I don't have too strong feelings about it and am okay if `getSlotOriginalClassNameProp` is preferred, but I don't find it super intuitive

Reply:
@CampbellOwen do you have another suggestion for the name?

Reply:
To be honest I like your original `getSlotClassNameProp` 😄. I don't think it's a big deal either way though

Reply:
i like short and sweet plus docs to explain. I think almost any name we pick will need some explanation.

Reply:
https://justflipacoin.com/

Reply:
> https://justflipacoin.com/

Alright it's decided, we're going to name it `HEADS`. Or actually, `getSlotClassNameProp_unstable`.